atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -68,7 +76,69 @@ static synchronized void setLogArbitrator(KeyValueServiceLogArbitrator arbitrato
                 logArbitrator.isColumnNameSafe(tableReference, columnName));
     }
 
+    public static Arg<Long> duration(Stopwatch stopwatch) {
+        return getArg("duration", stopwatch.elapsed(TimeUnit.MILLISECONDS), true);
+    }
+
+    public static Arg<String> method(String method) {
+        return getArg("method", method, true);
+    }
+
+    public static Arg<Integer> cellCount(int cellCount) {
+        return getArg("cellCount", cellCount, true);
+    }
+
+    public static Arg<Integer> tableCount(int tableCount) {
+        return getArg("tableCount", tableCount, true);
+    }
+
+    public static Arg<Integer> rowCount(int rowCount) {
+        return getArg("tableCount", rowCount, true);
+    }
+
+    public static Arg<Long> sizeInBytes(long sizeInBytes) {
+        return getArg("sizeInBytes", sizeInBytes, true);
+    }
+
+    public static Arg<Integer> columnCount(ColumnSelection columnSelection) {
```

Comment:
could this method do the logic for specifying `"all"` vs a count? seems off to have a function called `allColumns` below that actually returns a count